modules/041-linstor/templates/nodegroupconfiguration-configure-lvm.yaml

```diff
@@ -59,5 +59,40 @@ spec:
     EOF
     }
     
-    configure_lvm
+    configure_lvm_use_lvmetad() {
+      command -V lvmconfig >/dev/null 2>&1 || return 0
+      test -f /etc/lvm/lvm.conf || return 0
+      current_use_lvmetad=$(lvmconfig global/use_lvmetad 2>/dev/null || true)
+    
+      case "${current_use_lvmetad}" in
+        '' ) return 0 ;;
+        'use_lvmetad=0'*) return 0 ;;
+        'use_lvmetad=1'*) ;;
+        *) echo error parsing use_lvmetad >&2; return 1 ;;
+      esac
+    
+      lvmconfig --config "global/use_lvmetad=0" --withcomments --merge > /etc/lvm/lvm.conf.$$
+      mv /etc/lvm/lvm.conf.$$ /etc/lvm/lvm.conf
+    }
+    
+    disable_lvmetad_service() {
+      is_lvm2_lvmetad_service_enabled=$(systemctl is-enabled lvm2-lvmetad.service 2>/dev/null)
+      if [[ $is_lvm2_lvmetad_service_enabled != "masked" && $is_lvm2_lvmetad_service_enabled ]] ; then
+        systemctl stop lvm2-lvmetad.service >/dev/null 2>&1
+        systemctl disable lvm2-lvmetad.service >/dev/null 2>&1
+        systemctl mask lvm2-lvmetad.service >/dev/null 2>&1
+      fi
+
+      is_lvm2_lvmetad_socket_enabled=$(systemctl is-enabled lvm2-lvmetad.socket 2>/dev/null)
+      if [[ $is_lvm2_lvmetad_socket_enabled != "masked" && $is_lvm2_lvmetad_socket_enabled ]]; then
+        systemctl stop lvm2-lvmetad.socket >/dev/null 2>&1
+        systemctl disable lvm2-lvmetad.socket >/dev/null 2>&1
+        systemctl mask lvm2-lvmetad.socket >/dev/null 2>&1
+      fi
+    }
+    
+    
+    configure_lvm_global_filter
+    configure_lvm_use_lvmetad
```

Comment:
Someone would understand that we `configure` `lvm` to `use` `lvmetad` but we actually do it oposit way, please consider some naming like:

- `configure_lvm_use_no_lvmetad`
- `configure_lvm_disable_lvmetad`
- `disable_lvmetad_lvm_conf`

Reply:
Fixed